source/contribute.rst

```diff
@@ -41,128 +41,170 @@ contributing back to a Doctrine project.
 Initial Setup
 -------------
 
--  Setup a `GitHub <https://github.com>`_ account.
--  Fork the repository of the project you want to contribute to. In this example
-   it will be `DBAL <https://github.com/doctrine/dbal>`_
--  Clone your fork locally
+We are assuming you have cloned the project, have a Github account, and
+have forked the project. If any of this sounds unfamiliar, please `learn
+about that
+<https://docs.github.com/en/get-started/quickstart/contributing-to-projects>`_
+first.
+
+When cloning, you will end up with a remote called ``origin``. This can
+be confusing because sometimes, people use ``origin`` to refer to their
+fork.
+
+To avoid confusion, we recommend you to rename the ``origin`` remote to
+``doctrine``:
 
 .. code-block:: console
 
-    $ git clone git@github.com:username/dbal.git
+    $ git remote rename origin doctrine
 
--  Enter the dbal directory and add the **doctrine** remote
+Also, please ensure your Github fork is known as ``fork`` locally:
 
 .. code-block:: console
 
-    $ cd dbal
-    $ git remote add doctrine git://github.com/doctrine/dbal.git
+    $ git remote add fork git@github.com:<your-name-here>/orm.git
```

Comment:
Suggestion for this entire section:

---

To make the example commands given in the rest of this page less ambiguous, we will add two explicit names for remote Git repositories: `doctrine` for the original Doctrine project you're contributing to, and `fork` for your fork of it.

.. code-block:: console
    $ [`cd` into the directory created by `git clone ...`]
    $ git remote add doctrine git@github.com:doctrine/<name-of-project>.git
    $ git remote add fork git@github.com:<your-github-name>/<name-of-project>.git